include/extractor/extraction_containers.hpp

```diff
@@ -84,11 +83,12 @@ class ExtractionContainers
 
     std::vector<InputManeuverOverride> external_maneuver_overrides_list;
     std::vector<UnresolvedManeuverOverride> internal_maneuver_overrides;
+    std::unordered_set<NodeID> internal_barrier_nodes;
```

Comment:
The same about naming.

Reply:
Renamed to `used_barrier_nodes` as we already use `barrier_nodes` for barrier's OSM ids.